backends-velox/src/test/scala/org/apache/gluten/execution/VeloxAggregateFunctionsSuite.scala

```diff
@@ -428,6 +428,25 @@ abstract class VeloxAggregateFunctionsSuite extends VeloxWholeStageTransformerSu
     }
   }
 
+  testWithSpecifiedSparkVersion("regr_sxy", Some("3.4")) {
```

Comment:
Is this agg function not supported in other Spark version? Maybe you can consider using `isSparkVersionAtleast` to control the test coverage.

Reply:
Perhaps `isSparkVersionAtleast ` is same as `testWithSpecifiedSparkVersion`.

Reply:
Yes, they are equivalent. `testWithSpecifiedSparkVersion` looks more clear to reader.

Reply:
You guys are correct. Seems `isSparkVersionAtleast` is not necessary and I see there are some usage. Let me do a clean up as a follow up. Thanks.